go/pkg/client/cas_test.go

```diff
@@ -1221,6 +1221,195 @@ func TestDownloadActionOutputs(t *testing.T) {
 	}
 }
 
+func TestDownloadActionOutputsWithWorkingDir(t *testing.T) {
```

Comment:
It looks like this is a copy of `TestDownloadActionOutputs`. I don't think this is necessary. There was no functionality change on `cas.go`. Might as well just tweak it to use `wd` as the `outDir` or try different `outDirs`.